tools/generate-helpers/pg-table-bindings/list.go

```diff
@@ -54,6 +54,7 @@ func init() {
 		&storage.ProcessBaselineResults{}:                       resources.ProcessWhitelist,
 		&storage.ProcessBaseline{}:                              resources.ProcessWhitelist,
 		&storage.ProcessIndicator{}:                             resources.Indicator,
+		&storage.ResourceCollection{}:                           resources.Image, // TODO change this
```

Comment:
Recommend adding  `Workflow Administration` resource first and putting the correct value here from the very beginning.